### DISCUSSION TITLE
set the scope

### DISCUSSION THREAD

**Discussion by ucxl** [LSPosed/LSPosed#1527] [Q&A] [answered]
Can I provide an interface for my module to set the scope

For example: setmodulescope (string packagename, set < Application > Applications)

**Comment by yujincheng08** [comment, marked as answer]
#9

**Reply by ucxl** [reply]
I really don't understand how to use it  :  https://github.com/Xposed-Modules-Repo/XposedService

Is there a code example？

I just need to add and delete scopes

So for the root framework, putting the binder to a static field of modules' classloader is a stable way. And getService can directly return the binder. ----I don't understand what this sentence means

**Reply by yashikada** [reply]
you can't use it, because is incomplete. I tried without success. This feature has many requests but is hard todo, is old about one year
